misc/cleanup-dash.py

```diff
@@ -121,20 +121,19 @@ def set_hide_controls(dashboard):
 def set_unique_ids(dashboard):
     """To avoid the most common merge error: duplicate row ids."""
     ids = set()
-    for row_index, row in enumerate(dashboard['rows']):
-        for panel_index, panel in enumerate(row['panels']):
-            id_ = panel['id']
-            if id_ in ids:
-                id_ = 1
-                while id_ in ids:
-                    id_ += 1
+    for panel_index, panel in enumerate(dashboard['panels']):
+        id_ = panel['id']
+        if id_ in ids:
+            id_ = 1
+            while id_ in ids:
+                id_ += 1
 
-                print('ID #%s was not unique - changed to #%s' % (
-                    panel['id'], id_
-                ))
-                dashboard['rows'][row_index]['panels'][panel_index]['id'] = id_
+            print('ID #%s was not unique - changed to #%s' % (
+                panel['id'], id_
+            ))
+            dashboard['panels'][panel_index]['id'] = id_
```

Comment:
looks like an indentation has been lost here. now the loop does nothing except incrementation.